crates/control/src/localization_recorder.rs

```diff
@@ -15,7 +15,9 @@ use types::{
     game_controller_state::GameControllerState, line_data::LineData, primary_state::PrimaryState,
 };
 
+#[derive(Deserialize, Serialize)]
 pub struct LocalizationRecorder {
```

Comment:
Can we remove this node now?

Reply:
Yesterday I added this to Ideas for Next Iteration because the localization recording is a fully working feature (with replay in Twix) but recording alone is not feature-complete since replay is still missing. Once a basic version of replay is working, we can delete it.